src/game/common/globaldata.h

```diff
@@ -192,12 +192,7 @@ class GlobalData : public SubsystemInterface
     Coord3D m_terrainLightPos[LIGHT_COUNT];
     float m_infantryLight[TIME_OF_DAY_COUNT];
     float m_infantryLightOverride; // Based on ScriptAction::doSetInfantryLightingOverride
-    float m_easySoloHumanHealthBonus;
-    float m_normalSoloHumanHealthBonus;
-    float m_hardSoloHumanHealthBonus;
-    float m_easySoloAIHealthBonus;
-    float m_normalSoloAIHealthBonus;
-    float m_hardSoloAIHealthBonus;
+    float m_soloHealthBonus[2][3];
```

Comment:
We need named values for this. With plain numbers this is unreadable.

Reply:
After some more thinking and some discussion with toms, I came up with something for this.